test-utils/src/bitcoin_manager.rs

```diff
@@ -28,54 +26,77 @@ impl<'a> BitcoinManager<'a> {
     ) -> Result<BitcoinManager<'a>> {
         let p2p_port = get_available_port()?;
         let rpc_port = get_available_port()?;
+        let storage_dir = output_dir
+            .path()
+            .join(&format!("bitcoind_{}", settings.node_id));
+        std::fs::create_dir(&storage_dir)?;
 
-        let manager = Manager::new(output_dir, "bitcoind", &settings.node_id)?;
```

Comment:
So you don't want to use the the Manager struct at all? Do you think there is not enough generic functionality for it?

Reply:
We can simplify the following question if we do not care and the test and the CI flow are happy.  But for now, I think it should be refactored.

Does every service need pipe on stdin/stdout? 
Is the log interface for every service missing?
Are all services with the same log interface?
Does every service use the same retry logic?